docker-bake.hcl

```diff
@@ -25,7 +29,12 @@ group "default" {
 
 target "server" {
   dockerfile = "server.Dockerfile"
-  tags = ["temporaliotest/server:${IMAGE_TAG}", "temporaliotest/server:latest"]
+  # When building locally always tag latest. These will be overridden in CI to only
+  # conditionally tag the latest branch
```

Comment:
I don't get this comment.

Reply:
That was left in from an earlier attempt and is no longer accurate. I’ll remove it shortly